portfolio/src/main/webapp/script.js

```diff
@@ -269,3 +269,38 @@ function buildInfoWindowInput(lat, lng) {
 
   return containerDiv;
 }
+
+async function getLoginStatusUsingAsyncAwait() {
+
+  var section = document.getElementById('login-container');
+
+  fetch('/login').then(response => response.json()).then((status) => {
+    // status is an object, not a string, so we have to
+    // reference its fields to create HTML content
+
+    const statusListElement = document.getElementById('login-container');
```

Comment:
This seems to be the same as "section" - is that intentional? Is it supposed to be the same element?